rabbitmq/README.md

```diff
@@ -55,6 +59,106 @@ To configure this check for an Agent running on a host:
 
 2. [Restart the Agent][8].
 
+##### Migrating from Management Plugin to Prometheus Plugin.
```

Comment:
Let's move this section under `Preparing RabbitMQ`, above the management plugin instructions.

```suggestion
##### RabbitMQ Prometheus Plugin
```
We can put the remap table in a new section at the bottom